k-distribution/tests/regression-new/issue-3647-debugTokens/a.test.kast.out

```diff
@@ -1,12 +1,12 @@
 |"Match"                                                                          | (location)    | Terminal            |
 |---------------------------------------------------------------------------------|---------------|---------------------|
-|"1"                                                                              | (1,1,1,2)     | r"[0-9]+"           |
+|"1"                                                                              | (1,1,1,2)     | r"[\\+-]?[0-9]+"    |
```

Comment:
The issue here is that signed and unsigned integers now have overlapping lexical syntax at the same priority, right? (that is, this particular instance of instability will get resolved for free when you come round to implementing the overlap checker?)

Reply:
Yup! This will be stabilized with the overlap changes